src/main/java/org/jabref/gui/maintable/SmartConstrainedResizePolicy.java

```diff
@@ -14,9 +14,9 @@
 import org.slf4j.LoggerFactory;
 
 /**
- * This resize policy is almost the same as {@link TableView#CONSTRAINED_RESIZE_POLICY}
+ * This resize policy is almost the same as {@link TableView#CONSTRAINED_RESIZE_POLICY_FLEX_LAST_COLUMN}
```

Comment:
This is wrong.

Did you read the text around?

Did you follow the links?

Sorry, please revert this!

The constraint policies are different. SmartConstrainedResizePolicy code stayed the same - and `CONSTRAINED_RESIZE_POLICY_FLEX_LAST_COLUMN` is different from `CONSTRAINED_RESIZE_POLICY`

Reply:
Sorry I checked the link TableView#CONSTRAINED_RESIZE_POLICY, it says it is deprecated and use CONSTRAINED_RESIZE_POLICY_FLEX_LAST_COLUMN instead. So I thought I can fix the comment to avoid @deprecation issue, I'll revert it.